tests/lwip/ip.c

```diff
@@ -64,7 +72,11 @@ static void *_server_thread(void *args)
             char addrstr[IPV6_ADDR_MAX_STR_LEN];
 
             printf("Received IP data from [%s]:\n",
+#if MODULE_LWIP_IPV6
                    ipv6_addr_to_str(addrstr, (ipv6_addr_t *)&src.addr.ipv6,
+#else
+                   ipv4_addr_to_str(addrstr, (ipv4_addr_t *)&src.addr.ipv4,
+#endif
                                     sizeof(addrstr)));
```

Comment:
This should also be in both `#ifdef` blocks (also other files).

Reply:
Not really clear what you mean :worried:

Reply:
While the `sizeof(addrstr)));` bit is shared by both blocks, it's still cleaner not to split the function like that.

Reply:
It is in general not only bad style but also not a good idea to split up a C-state by a pre-processor statement like that.

Reply:
Should the `printf` then also be in both `#ifdef` blocks?

Reply:
Yes, I'd say so. Alternatively, to avoid more code duplication I just would pull out the string conversion out of the print.

Reply:
Ok, done.